build.sbt

```diff
@@ -1,12 +1,12 @@
 name := "spark-essentials"
 
-version := "0.1"
+version := "0.2"
 
-scalaVersion := "2.12.10"
+scalaVersion := "2.13.12"
 
-val sparkVersion = "3.2.1"
-val vegasVersion = "0.3.11"
-val postgresVersion = "42.2.2"
+val sparkVersion = "3.5.0"
+val postgresVersion = "42.6.0"
+val log4jVersion = "2.20.0"
```

Comment:
These are the very latest versions to upgrade.

Reply:
Nice 👍